helm/secrets-provider/values.schema.json

```diff
@@ -53,7 +53,7 @@
         "tag": {
           "type": ["string", "number"],
           "minLength": 1,
-          "pattern": "(^\\d+(\\.\\d+){0,2}|dev)$"
+          "pattern": "(^\\d+(\\.\\d+){0,2}|dev|latest)$"
```

Comment:
I'm against using the `latest` label. We added `dev` for supporting the tests. Why do you need it?

Reply:
we need this latest tag for testing. A lot of places in the scripts use latest (our unit tests, integration tests, etc) so I want to add it in

Reply:
for what do we need dev ?

Reply:
dev removed